src/app/pages/SearchResultsPage/GlobalSearchResultsView.tsx

```diff
@@ -33,7 +33,7 @@ export const GlobalSearchResultsView: FC<{
   const otherNetworks = RouteUtils.getEnabledNetworks().filter(isNotMainnet)
   const notificationTheme = themes[Network.testnet]
   const mainnetResults = searchResults.filter(isOnMainnet).sort(orderByLayer)
-  const otherResults = searchResults.filter(isNotOnMainnet).sort(orderByLayer)
+  const otherResults = searchResults.filter(isNotOnMainnet).filter(isNotOnHiddenLayer).sort(orderByLayer)
```

Comment:
Search still always makes requests for Ponuts-X results
(remove `.replace('/v1/pontusx', '/v1/sapphire')` to see them)

Reply:
This is being solved in #1280.